apps/projects/contracts/Projects.sol

```diff
@@ -376,7 +380,7 @@ contract Projects is AragonApp, DepositableStorage {
     }
 
     /**
-     * @notice Remove repository from the Projects app
+     * @notice Remove repository `_repoId`
```

Comment:
Same thing here. Do we have some way to display the name of the repo at this point? Would we have to send extra data to the contract, which it wouldn't actually need to use?

Reply:
I was asking kevin if we can add metadata for the repo @topocount - but if it's IPFS we still have ugly hash until we develop an IPFS radspec resolver